src/containers/App/index.js

```diff
@@ -30,55 +30,55 @@ function App(props) {
-
-      `${window.location.host}/api/storage/h5p/libraries/H5P.Question-1.4/scripts/question.js?ver=1.4.7`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.Question-1.4/scripts/explainer.js?ver=1.4.7`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.Question-1.4/scripts/score-points.js?ver=1.4.7`,
-      `${window.location.host}/api/storage/h5p/libraries/Drop-1.0/js/drop.min.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.Transition-1.0/transition.js?ver=1.0.4`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-help-dialog.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-message-dialog.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-progress-circle.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-simple-rounded-button.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-speech-bubble.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-throbber.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-tip.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-slider.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-score-bar.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-progressbar.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.JoubelUI-1.3/js/joubel-ui.js?ver=1.3.9`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebra3d-1.0/scripts/deployggb.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebra3d-1.0/scripts/geogebra.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraClassic-1.0/scripts/deployggb.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraClassic-1.0/scripts/geogebra.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraGeometry-1.0/scripts/geogebra.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraGeometry-1.0/scripts/deployggb.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraGraphing-1.0/scripts/deployggb.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraGraphing-1.0/scripts/geogebra.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraIM68Math-1.0/scripts/deployggb.js?ver=1.0.2`,
-      `${window.location.host}/api/storage/h5p/libraries/H5P.GeoGebraIM68Math-1.0/scripts/geogebra.js?ver=1.0.2`,
+      `${process.env.REACT_APP_RESOURCE_URL}/storage/h5p/h5p-core/js/jquery.js`,
```

Comment:
Please use config variable

Reply:
config variables are not working server, tested before